ReactSkia/pluginfactory/plugins/stub/PlatformManager.h

```diff
@@ -14,6 +14,11 @@ namespace rnsplugin {
 
 using namespace std;
 
+using DeviceCPUInfo = RNSPlatformManagerInterface::PlatformDevice::DeviceProcessUintInfo;
```

Comment:
I guess it means "DeviceProcessUnitInfo". If you are changing, change it all over